src/tls.c

```diff
@@ -4488,7 +4489,7 @@ static word16 TLSX_Cookie_GetSize(Cookie* cookie, byte msgType)
     if (msgType == client_hello || msgType == hello_retry_request)
         return OPAQUE16_LEN + cookie->len;
 
-    return SANITY_MSG_E;
+    return (word16) SANITY_MSG_E;
```

Comment:
This isn't a valid return code / cast since SANITY_MSG_E is -394 (and word16 is unsigned). In fact the handling of this in CKE_GET_SIZE (TLSX_Cookie_GetSize) is wrong. Looks like its not handled correctly in current master.
@moisesguimaraes , can you look at the handling for the return codes on `TLSX_Cookie_GetSize`, `TLSX_Cookie_Write` and `TLSX_KeyShare_GetSize`?